gargoyle-buildrpm.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/sh
```

Comment:
This script uses some non-POSIX stuff (e.g. `pushd`), so this should be `/bin/bash`.

Reply:
Will do.

Reply:
Checked in changes.